CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 All notable changes to `dash` will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+## [2.3.2]
```

Comment:
```suggestion
## [Unreleased]
```
We'll decide on the version number later, but this will be at least 2.4 since this is a new feature :)

Reply:
Ahhh true! Will fix this in the next commit